src/Raven.Studio/typescript/common/abstractWebSocketClient.ts

```diff
@@ -1,11 +1,11 @@
 /// <reference path="../../typings/tsd.d.ts" />
-
 import database = require("models/resources/database");
 import appUrl = require("common/appUrl");
 
 abstract class abstractWebSocketClient<T> {
 
     private static readonly readyStateOpen = 1;
+    connectionOpened = ko.observable<boolean>(false);
```

Comment:
can we name it connected or isConnected?

Reply:
done - renamed to `isConnected`